src/Homepage/Homepage.js

```diff
@@ -9,20 +9,29 @@ export class Homepage extends Component {
     render(){
 
         return(
-        <> 
         <nav className="nav-bar">
-          <h1 className="main-title">Rancid Tomatillos</h1>
-          <button className="Login">Login</button>
+          <h1 className="main-title"><i className="fas fa-seedling"></i>Rancid Tomatillos</h1>
+
+          <div className="search-container">
+            <i className="fas fa-search"></i>
+            <label htmlFor="search-bar">
+            <input
+              className="search-bar"
+              type="text" 
+              className="search-bar" 
+              id="searh-bar"/>
```

Comment:
I'm not 100% we need a search bar? That might have been an extension. That being said, I think we are making nice enough progress to be able to tackle that either way, and I think it would be a good addition as well.

Reply:
I think if we are able to get to the search bar functionality together it will be a great addition to the project. 👍